src/cookbook/effects/download-button.md

```diff
@@ -76,13 +79,14 @@ class DownloadButton extends StatelessWidget {
     ),
   }) : super(key: key);
 
- final DownloadStatus status;
- final Duration transitionDuration;
+  final DownloadStatus status;
+  final Duration transitionDuration;
 
- Widget build(BuildContext context) {
-   // TODO:
-   return SizedBox();
- }
+  @override
+  Widget build(BuildContext context) {
+    // TODO:
```

Comment:
Please add a couple of words to the todo line =)

Reply:
Done!